code/game/gamemodes/objective.dm

```diff
@@ -697,6 +726,11 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
 	return target
 
 /datum/objective/destroy/update_explanation_text()
+	if(delayed_objective)
+		explanation_text = "Your objective is to either destroy or steal an Artificial Intelligence. Please hold for further details."
+		delayed_objective = FALSE
```

Comment:
there has to be a better place to set `delayed_objective = FALSE`

Reply:
Changed